FlaskApp/site_utils.py

```diff
@@ -0,0 +1,9 @@
+import string
+
+def make_url_slug(title):
```

Comment:
You should add some documentation to this function. Someone looking at this code for the first time might not immediately understand what's going on, or the purpose of this function. 

Take a look at this, don't need to read the whole thing but it should give you the general gist:
https://www.python.org/dev/peps/pep-0257/